### ISSUE TITLE
Up for adoption?

### ISSUE THREAD

**Issue by jrjohnson** [simonexmachina/ember-cli-sass#203]
When an original owner no longer has time to maintain the addon, there is a place where the addon can be "put up for adoption." Melanie Sumner has created [adopted-ember-addons](https://github.com/adopted-ember-addons).

@aexmachina - would you consider allowing us to adopt this addon so that it may continue to provide useful service to the community?

**Comment by aexmachina** [comment]
Yes, I think that makes sense @jrjohnson. Please let me know how you'd like to proceed.

**Comment by jrjohnson** [comment]
@aexmachina it looks like the process is outlined at https://github.com/adopted-ember-addons/program-guidelines and there is help available in #adopted-ember-addons in discord. I'm happy to help, but it looks like the owner probably has to complete most of the steps.

**Comment by Alonski** [comment]
@aexmachina I am able to help with the transfer as well as @knownasilya

**Comment by jrjohnson** [comment]
@simonexmachina checking in on this. Can you get in contact with `@Alonski` and arrange the transfer of ownership?

**Comment by simonexmachina** [comment]
I thought I'd already done this, my apologies. @Alonski I'll add you as a collaborator, and I'll complete the transfer once it's ready. Thanks for your help! :)

**Comment by Alonski** [comment]
Thanks @simonexmachina.
Are you planning on finishing the transfers? 
BTW thanks for all the great work you've done!

**Comment by simonexmachina** [comment]
As I understand, there are some steps that need to be done to the code
first to get it ready. See the adopted add-ons repo above for more
information

>

**Comment by Alonski** [comment]
@simonexmachina Yes :) My question is are you planning to do the necessary changes or should I? I'd love of you had the time to do it as I'm currently a bit busy with finals 😅

**Comment by simonexmachina** [comment]
I thought you were offering. I'm not able to do it until next week at the earliest. I'm not even using Ember any more, but I'll try to hand it over to the community. If you find the time please go ahead, but otherwise I'll do it.

**Comment by Alonski** [comment]
I'll see what I can do. One of us will find the time 😁
BTW I don't think I have admin status so I can only do so much

**Comment by Alonski** [comment]
@simonexmachina Is this repo ready to transfer ownership?

**Comment by simonexmachina** [comment]
Yes it is, just waiting on @MelSumner to perform the steps from her end.
